src/SldStyleParser.ts

```diff
@@ -365,7 +365,8 @@ class SldStyleParser implements StyleParser {
           lineSymbolizer.cap = value;
           break;
         case 'stroke-dasharray':
-          lineSymbolizer.dasharray = value;
+          const dashStringAsArray = value.split(',').map((a: string) => parseFloat(a));
```

Comment:
Insn't the dash-array modeled like 

`<CssParameter name="stroke-dasharray">1 0</CssParameter>`

in SLD?

Reply:
Yes but the xml2js gives me a comma seperated string from this... I'll double check it and will give feedback here afterwards.

Reply:
You were totally right. Strange that my code actually worked in the geostyler. I added a dasharray to the test files to be sure.
Thx